src/components/dev-hub/blog-post-title-area.js

```diff
@@ -18,26 +19,54 @@ const PostMetaLine = styled('div')`
     }
 `;
 
+const bulletStyling = css`
+    @media ${screenSize.xSmallAndUp} {
+        &:before {
+            content: '\u2022';
+            margin-right: ${size.tiny};
+        }
+    }
+`;
+
 const DateText = styled(P)`
-    margin-right: ${size.medium};
+    margin-right: ${size.tiny};
     @media ${screenSize.upToLarge} {
         font-size: ${fontSize.xsmall};
     }
+    ${({ withBullet }) => withBullet && bulletStyling};
```

Comment:
Bet we could do this with last-child or the `+` selector, but mostly just a noit

Reply:
Only issue there is if the updated date is not provided we don't want the bullet on the initial published date. I would like to keep as-is?

Reply:
Sure you can keep, but that can be done with css :)